app/controllers/catalog_controller.rb

```diff
@@ -6,7 +6,6 @@ class CatalogController < ApplicationController
 
   # These before_filters apply the hydra access controls
   before_filter :enforce_show_permissions, only: :show
-  skip_before_filter :default_html_head
```

Comment:
??

Reply:
that method no longer exists. I believe it was generated code.

Reply:
thank you!